src/installer/tests/TestUtils/Assertions/CommandResultAssertions.cs

```diff
@@ -21,121 +21,121 @@ public CommandResultAssertions(CommandResult commandResult)
         public AndConstraint<CommandResultAssertions> ExitWith(int expectedExitCode)
         {
             Execute.Assertion.ForCondition(Result.ExitCode == expectedExitCode)
-                .FailWithPreformatted($"Expected command to exit with {expectedExitCode} but it did not.{GetDiagnosticsInfo()}");
+                .FailWith($"Expected command to exit with {expectedExitCode} but it did not.{GetDiagnosticsInfo()}");
```

Comment:
I thought we had to explicitly use our extension and `AddFailure` (or I guess `AddPreFormattedFailure` in newer versions) because `FailWith` would always go through the the `FluentAssertions` formatter which messed with things like newlines. Does the new version not do that anymore?

Reply:
There were a lot of API changes between v4 and the latest v6. AssertionScope.Succeeded etc. are not available on public surface anymore.

> `FailWith` would always go through the the FluentAssertions formatter which messed with things like newlines.

If this is still a problem, we can change the behavior with `AssertionScope.FormattingOptions.UseLineBreaks = true` (or fluent API `.UseLineBreaks`).

Reply:
There were a lot of API changes between v4 and the latest v6. AssertionScope.Succeeded etc. are not available on public surface anymore.

> `FailWith` would always go through the the `FluentAssertions` formatter which messed with things like newlines.

If this is still a problem, we can change the behavior with `AssertionOptions.FormattingOptions.UseLineBreaks = true` (or fluent API `.UsingLineBreaks`).